tests/test_wave_01.py

```diff
@@ -51,22 +51,22 @@ def test_get_task(client, one_task):
     }
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
+# @pytest.mark.skip(reason="No way to test this feature yet")
 def test_get_task_not_found(client):
     # Act
     response = client.get("/tasks/1")
     response_body = response.get_json()
 
     # Assert
     assert response.status_code == 404
-
-    raise Exception("Complete test with assertion about response body")
+    assert response_body == {"message" : "Task 1 does not exist"}
```

Comment:
Nice work with this test completion! This assertion is a great way to validate functionality is working as expected